feat(ktooltip): reskin component styles [KHCP-9007] (#1988)
Kong/kongponents: docs/components/tooltip.md

# Tooltip

> A tooltip is a brief, informative message that appears when a user hovers over, focuses on, or taps an element, providing additional context, guidance, or information without cluttering the interface.

Reviewer: ```suggestion
A tooltip is aninformative message that appears when a user hovers over, focuses on, or taps an element, providing additional context, guidance, or information without cluttering the interface.
```

<KTooltip text="Simple tooltip.">
  <InfoIcon />
</KTooltip>

```html
<KTooltip text="Simple tooltip.">
  <InfoIcon />
</KTooltip>
```

## Props

### text

Text to display in the tooltip.

> <KTooltip text="You will recieve a notification once your request is approved.">

Reviewer: spelling
```suggestion
<KTooltip text="You will receive a notification once your request is approved.">
```

  <KButton>Request</KButton>
</KTooltip>

```html

> <KTooltip text="You will recieve a notification once your request is approved.">

Reviewer: ```suggestion
<KTooltip text="You will receive a notification once your request is approved.">
```

  </li>
</ul>

<div class="tooltip-container">
  <KTooltip placement="bottom" text="A tooltip that appears on the bottom.">
    <KButton>bottom</KButton>
  </KTooltip>
  <KTooltip placement="topEnd" text="A tooltip that appears on the top.">
    <KButton>topEnd</KButton>
  </KTooltip>
  <KTooltip placement="left" text="A tooltip that appears on the left.">
    <KButton>left</KButton>
  </KTooltip>
  <KTooltip placement="bottomStart" text="A tooltip that appears on the bottom.">
    <KButton>bottomStart</KButton>
  </KTooltip>
</div>

```html
<KTooltip placement="bottom" text="A tooltip that appears on the bottom.">
  <KButton>Sample Button</KButton>
</KTooltip>
```

### positionFixed

> Use fixed positioning of the popover to avoid content being clipped by parental boundaries - defaults to `false`. See [KPop docs](/components/popover#positionfixed) for more information.

Reviewer: question: any reason we default this to false instead of true?

Author: Changing to true

Reviewer: This would be a breaking change. I'm asking, should we

Author: Based on discussion from the other thread yeah, we should
